supplementary_style_guide/glossary_terms_conventions/general_conventions/v.adoc

```diff
@@ -202,6 +202,17 @@ Although the _IBM Style Guide_ recommends using "diskette drive" instead of "flo
 
 *See also*:
 
+[discrete]
+[[volatile-storage]]
+==== image:images/yes.png[yes] volatile storage (noun)
+*Description*: Temporary storage, for example random access memory (RAM), that requires power to maintain the stored information. The stored data is lost when the device is turned off or restarts.
```

Comment:
To make this a complete sentence you could say something like:

`Volatile storage is a temporary type of storage, for example, random access memory (RAM) that requires power to maintain the stored information. The stored data is lost when the device is turned off or restarts.`